docs/user/dql/functions.rst

```diff
@@ -2728,6 +2727,30 @@ Examples::
     | 2000-01-18 00:00:00                            | 1999-10-01 00:00:00                                |
     +------------------------------------------------+----------------------------------------------------+
 
+TIMESTAMPDIFF
+-------------
+
+Description
+>>>>>>>>>>>
+
+Usage: Returns a LONG representing the difference in time for the input arguments arguments. It takes an INTERVAL argument which determine the unit of time used to measure the difference and two additional arguments for the start and end times.
```

Comment:
Why don't we just say:
```
Return type: LONG
```

Reply:
Maybe something shorter like: 
```
TIMESTAMPDIFF(interval, start, end) returns the different between the start and end date/times in interval units. 
```

Reply:
Does it round up/down?  
Example, if I give it two times, and ask for YEAR interval, will it return 0?

Reply:
> Why don't we just say:
> 
> ```
> Return type: LONG
> ```

Ah see because by making it longer whoever is reading it will lose interest and any mistakes in my description will go unnoticed. /s

I'll reword it to be shorter.

---

> Does it round up/down? Example, if I give it two times, and ask for YEAR interval, will it return 0?

It might round down, but never up. For it to return a non-zero value the difference between arguments must be greater than the given interval. 

Example:
(YEAR, '2000-01-01 00:00:00', '2000-01-01 00:30:00') -> 0
(MINUTE, '2000-01-01 00:00:00', '2000-01-01 00:30:00') -> 30 (same query as the first but different interval type)
(YEAR, '2000-01-01 00:00:00', '2000-12-31 00:30:00') -> 0 (Even though the difference is close to 365 days it is not a full year so it returns 0)

Reply:
If you give it two different years but the total difference in time is less than a year total e.g. '2000-02-01 00:00:00' and '2001-01-01 00:00:00' will it return a 0 or a 1?

Reply:
> 

It returns 0 since the difference is less than 1 year